src/model/table.py

```diff
@@ -1,30 +1,87 @@
 from model.table_metadata import *
+import time
 
 
 class Table:
-    def __init__(self, database, table):
-        self.table_schema = table
-        self.__path__ = os.path.join(database.get_path(), table[Keys.NAME])
+    def __init__(self, database,  table_name, table_schema = None):
+        self.__path__ = os.path.join(database.get_path(), table_name)
+        if table_schema is None:
+            table_schema = TableMetaData.load_table_schema(self.__path__, table_name)
+        self.table_schema = table_schema
         self.__table_metadata__ = TableMetaData(self)
 
     def serialize(self):
         os.makedirs(self.__path__, exist_ok=True)
         self.__table_metadata__.serialize()
 
+    def get_name(self):
+        return self.__table_metadata__.name
+
     def get_path(self):
         return self.__path__
+        
+    def get_primary_key(self):
+        return self.__table_metadata__.primary_key
 
     # Will be implemented later in the project
-    def set(self):
-        pass
+    def __serialize_row__(self, data):
+        primary_key = self.get_primary_key()
+        can_overwrite = eval(self.__table_metadata__.overwrite)
+        path = Table.__get_row_path__(self.__path__, data.get(primary_key))
```

Comment:
I would suggest it returns a primary_key value. rename it by __get_row_primary_key(data)